src/main/uml/class-diagrams/entity-model.uml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Diagram>
```

Comment:
Wow, is this generated automatically?

Reply:
kind of... I selected the entity classes, right click, Diagrams, Show Diagram. Then Exported it to a uml file.